mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -672,23 +672,25 @@
-        "MOM_Tip_General_10" "Tip: When in the air, pressing a movement key opposite to the direction of movement will rapidly decrease your speed." // Block57
-        "MOM_Tip_General_11" "Tip: Remember to have fun. Don't get frustrated trying to beat your times on a map!" // hex
-        "MOM_Tip_General_12" "Tip: Use 'mom_practice' to explore and test if you are stuck or lost on a map." // hex
-        "MOM_Tip_General_13" "Tip: Adjust your mouse senstivity to your liking and try to avoid changing it frequently." // amp
-        "MOM_Tip_General_14" "Tip: Crouching while mid-air raises your feet and lowers your head, but extends your jump distance." // Bela + BorkChops
-        "MOM_Tip_General_15" "Tip: Check out the Momentum Discord to talk about game modes, discuss announcements, and more!" // BorkChops
-        "MOM_Tip_General_16" "Tip: The command 'cl_yawspeed' controls the speed of your turn binds, if you have any." // Pretzl
-        "MOM_Tip_General_17" "Tip: Being comfortable with your keybinds and posture helps avoid gaming-related injuries." // hex
-        "MOM_Tip_General_18" "Tip: Some game modes may be daunting at first, but don't let this stop you from learning them!" // amp
-        "MOM_Tip_General_19" "Tip: In the air, don't hold your forward key to go forwards. Let your momentum take care of that - control your movement with strafe keys." // porkie
+        "MOM_Tip_General_8" "Tip: Higher tier maps are harder to complete, but they give you more XP upon completion!"
+        "MOM_Tip_General_9" "Tip: Remember to have fun. Don't get frustrated trying to beat your times on a map!" // hex
+        "MOM_Tip_General_10" "Tip: Use 'mom_practice' to explore and test if you are stuck or lost on a map." // hex
+        "MOM_Tip_General_11" "Tip: Adjust your mouse senstivity to your liking and try to avoid changing it frequently." // amp
+        "MOM_Tip_General_12" "Tip: Crouching while mid-air raises your feet and lowers your head - extending your jump distance." // Bela + BorkChops
+        "MOM_Tip_General_13" "Tip: Check out the Momentum Discord to talk about game modes, discuss announcements, and more!" // BorkChops
+        "MOM_Tip_General_14" "Tip: The command 'cl_yawspeed' controls the speed of your turn binds, if you have any." // Pretzl
+        "MOM_Tip_General_15" "Tip: Being comfortable with your keybinds and posture helps avoid gaming-related injuries." // hex
+        "MOM_Tip_General_16" "Tip: Some game modes may be daunting at first, but don't let this stop you from learning them!" // amp
+        "MOM_Tip_General_17" "Tip: In the air, don't hold your forward key to go forwards. Let your momentum take care of that - control your movement with strafe keys." // porkie
+        "MOM_Tip_General_18" "Tip: Playing maps outside of your comfort zone aids improvement." // change name
+        "MOM_Tip_General_19" "Tip: If a map is too dark, increase the brightness in the settings." //nevier
 
```

Comment:
A tip that was missed (by me) was one for ignoring teleports in practice mode when the jump key is held:
```suggestion
"MOM_Tip_General_20" "Tip: Map teleports can be ignored in practice mode by holding jump." // Hellbent

```